Lib/tarfile.py

```diff
@@ -2447,20 +2447,52 @@ def is_tarfile(name):
 
 def main():
     import argparse
+    compressions = {
+        # gz
```

Comment:
Likewise, I don’t see the added values of the comments here.

Reply:
Removed